### ISSUE TITLE
Configure the release process

### ISSUE THREAD

**Issue by rachel-fenichel** [google/blockly#7765]

**Comment by lizschwab** [comment]
@maribethb I don't have access to the linked core release process doc. Is this something you can grant me access to?

**Comment by maribethb** [comment]
No, I don't have it either. But we did implement a lot of that and the core release process is so automated now anyway that I don't think you need it. At the time this was written, we still had develop/master and were largely doing a 10 step release process by hand vs just running the github action.
